documental/filters.py

```diff
@@ -22,16 +22,17 @@ class DocumentalDocsFilter(rest_framework.FilterSet):
     """Django filter `models.DocumentalDocs` data.
 
     Filter:
-        * acao_id (int): action identifier to be filtered
+        * acao_id (int): action identifier to be filtered.
         * co_cr (list): filtering Regional Coordination using code.
         * co_funai (list): filtering Indigenou Lands using Funai code.
         * start_date (str): filtering start date.
         * end_date (str): filtering end date.
+        * map_year (list): filteringend years of the maps.
     """
     
-    acao_id = rest_framework.CharFilter(
+    acao_id = CharInFilter(
```

Comment:
PEP8 import errors, it might come from `app.MethodName`

Reply:
Not modified. The return of the Foreign key model is charfield.